d3b_utils/aws_bucket_contents.py

```diff
@@ -59,12 +64,14 @@ def fetch_bucket_obj_info(
             object for object in bucket_contents if object["Size"] > 0
         ]
 
+    if drop_path:
+        for object in bucket_contents:
+            _, _, object["Key"] = object["Key"].rpartition("/")
```

Comment:
I'm not as keen on the idea of modifying `Key. Dropping out the path to the object makes it... no longer the key?

Reply:
maybe the docstring should note that when you set `drop_path=True`, you lose the ability to pinpoint _where_ in a bucket a file is. User may then run in to an issue where you have two files with the same name. 

And if that is a possibility, I don't think we would want to return that in the function.

Reply:
agree. maybe drop_path should use a different column name

Reply:
we should drop `drop_path`.